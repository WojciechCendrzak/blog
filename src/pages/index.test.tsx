import renderer from 'react-test-renderer';
import HomePage from '.';

describe('HomePage', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<HomePage allPostsData={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
