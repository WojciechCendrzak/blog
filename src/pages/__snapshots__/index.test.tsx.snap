// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should match the snapshot 1`] = `
<div
  className="container"
>
  <header
    className="header"
  >
    <div
      style={
        Object {
          "boxSizing": "border-box",
          "display": "inline-block",
          "margin": 0,
          "maxWidth": "100%",
          "overflow": "hidden",
          "position": "relative",
        }
      }
    >
      <div
        style={
          Object {
            "boxSizing": "border-box",
            "display": "block",
            "maxWidth": "100%",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          role="presentation"
          src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQ0IiBoZWlnaHQ9IjE0NCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
          style={
            Object {
              "border": "none",
              "display": "block",
              "margin": 0,
              "maxWidth": "100%",
              "padding": 0,
            }
          }
        />
      </div>
      <img
        alt="[Your Name]"
        className="borderCircle"
        decoding="async"
        src="/_next/image?url=%2Fimages%2Fprofile.jpg&w=384&q=75"
        srcSet="/_next/image?url=%2Fimages%2Fprofile.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fprofile.jpg&w=384&q=75 2x"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
    </div>
    <h1
      className="heading2Xl"
    >
      [Your Name]
    </h1>
  </header>
  <main>
    <section
      className="headingMd"
    >
      <p>
        [Your Self Introduction]
      </p>
      <p>
        (This is a sample website - youâ€™ll be building a site like this in
         
        <a
          href="https://nextjs.org/learn"
        >
          our Next.js tutorial
        </a>
        .)
      </p>
    </section>
    <section
      className="headingMd padding1px"
    >
      <h2
        className="headingLg"
      >
        Blog
      </h2>
      <ul
        className="list"
      />
    </section>
  </main>
</div>
`;
